src/app/shared/components/header/header.component.ts

```diff
@@ -1,18 +1,17 @@
-import { Component, OnInit } from "@angular/core";
+import { Component } from "@angular/core";
 import { AuthService } from "@app/auth/services/auth.service";
+import { AuthenticationFacade } from "@app/store/authentication/authentication.facade";
 
 @Component({
   selector: "app-header",
   templateUrl: "./header.component.html",
   styleUrls: ["./header.component.scss"],
 })
-export class HeaderComponent implements OnInit {
+export class HeaderComponent {
   isAuthorised: boolean = false;
 
-  constructor(public authService: AuthService) {}
-  ngOnInit(): void {
-    this.authService.isAuthorized$.subscribe(
-      (isAuth) => (this.isAuthorised = isAuth)
-    );
-  }
+  constructor(
+    public authService: AuthService,
```

Comment:
Are you sure both are needed here? Also, I think the `<ng-content>` tags and content projection can be left behind by moving the username and login/logout buttons here.